kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/utils/DefaultEngineOfBatchSize2.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (2025) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.delta.kernel.defaults.utils
+
+import io.delta.kernel.defaults.engine.DefaultEngine
+
+import org.apache.hadoop.conf.Configuration
+
+trait DefaultEngineOfBatchSize2 extends AbstractTestUtils {
+
+  override lazy val defaultEngine = DefaultEngine.create(new Configuration() {
```

Comment:
IMO this single value doesn't warrant a whole new trait. e.g. I don't think any other values would go in this trait, right? seems very single-purpose?

Could we not just define `val engineWithBatchSize2` inside of `AbstractTestUtils` ? and then explicitly use that inside of the required tests?